recipes/enso_metrics/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "enso_metrics" %}
+{% set version = "v1.1.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/CLIVAR-PRP/ENSO_metrics/archive/refs/tags/{{ version }}.tar.gz
+    sha256: e933f373be1d793b801ed7155690fc4759a463da5931f18b5117ba9b259840b3    
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - cdms2
+    - cdutil
+    - genutil
+    - numpy
+    - scipy
+    - udunits2
+
+test:
+  imports:
+    - EnsoMetrics.EnsoCollectionsLib
+    - EnsoMetrics.EnsoComputeMetricsLib
+
+about:
+    home: https://github.com/CLIVAR-PRP/ENSO_metrics
+    license: BSD-3-Clause
+    license_file: LICENSE
```

Comment:
Your upstream license is conda-forge. It seems like this is probably not correct. 

https://github.com/CLIVAR-PRP/ENSO_metrics/blob/6b5a3c33ec89a5c6de147bf6dc03872a89b9da90/LICENSE#L2

Reply:
License has been updated to give copyright to CLIVAR: https://github.com/CLIVAR-PRP/ENSO_metrics/commit/11f113afa56b3a1fb303860d3533c0f4017746da